### PR TITLE
Accessibility Tweaks

### PR DESCRIPTION
**Pull request by stemount**
The comment below will show which these address.

### CODE REVIEW

**Comment by stemount** [comment]
I found that the PAR authority flow edit buttons had yet again stopped working. @jaffamonkey we really need some better coverage around this today. Can we discuss?

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsAddressForm.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ public function titleCallback() {
     $par_data_partnership = $this->getRouteParam('par_data_partnership');
     if ($par_data_partnership) {
       $par_data_organisation = current($par_data_partnership->getOrganisation());
-      $this->pageTitle = $par_data_organisation->get('organisation_name')->getString();
+      $this->pageTitle = 'Edit your business address';
```
Comment (stemount) [inline web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsAddressForm.php]: maybe need to wrap this in a if for a business condition

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsAddressForm.php [hunk 2]
```diff
@@ -103,38 +103,38 @@ public function buildForm(array $form, FormStateInterface $form_state, ParDataPa
...
       '#type' => 'textfield',
-      '#title' => $this->t('County'),
+      '#title' => $this->t('Enter your County'),
       '#default_value' => $this->getDefaultValues("county"),
     ];
 
     $form['country'] = [
       '#type' => 'select',
-      '#title' => $this->t('Nation'),
+      '#title' => $this->t('Select your Nation'),
```
Comment (stemount) [inline web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsAddressForm.php]: We need to do this side wide.

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsTradingForm.php [hunk 3]
```diff
@@ -65,16 +67,15 @@ public function retrieveEditableValues(ParDataPartnership $par_data_partnership
   public function buildForm(array $form, FormStateInterface $form_state, ParDataPartnership $par_data_partnership = NULL, $trading_name_delta = NULL) {
     $this->retrieveEditableValues($par_data_partnership, $trading_name_delta);
 
-    $form['intro'] = [
-      '#markup' => $this->t('Change the Trading Name of your organisation'),
-      '#prefix' => '<h2>',
-      '#suffix' => '</h2>',
+    $form['trading_name_fieldset'] = [
```
Comment (stemount) [inline web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsTradingForm.php]: We also should do this sidewide as well to tidy things up, real lack of `form-group` css class across the board.

**Comment by stemount** [comment]
I really regret leaving comments on this PR as they were more for other devs as notes.

**Comment by stemount** [comment]
the PR is ready to merge anyway.

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsAddressForm.php (hunk 1 above)
Comment (jon-edery) [reply web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsAddressForm.php]: Think we can remove all this logic now ->  $par_data_partnership = $this->getRouteParam('par_data_partnership');
    if ($par_data_partnership) {
      $par_data_organisation = current($par_data_partnership->getOrganisation());

**Comment by jon-edery** [comment]
Minor comment looks like we have some validation logic we can remove as we are no longer using it part from that all looks good

**Comment by stemount** [comment]
@nao360 @kalpaitch is right, we are missing the tickets to make "Enter the full name" "select from the list" esque field labels for the other journeys.

**Pull request merged**